common/beerocks/bwl/dwpal/mon_wlan_hal_dwpal.cpp

```diff
@@ -739,6 +739,30 @@ bool mon_wlan_hal_dwpal::sta_beacon_11k_request(const SBeaconRequest11k &req, in
         cmd += " ssid=" + req_ssid;
     }
 
+    // use ap_ch_report in the request.
+    if (req.use_optional_ap_ch_report) {
+        std::stringstream ch_report_ss;
+        uint8_t current_channel = 0;
+        for (; current_channel < req.use_optional_ap_ch_report - 1; ++current_channel) {
+            ch_report_ss << +req.ap_ch_report[current_channel] << ",";
```

Comment:
(nitpick) the rest of this file uses the pattern
```suggestion
            cmd += std::to_string(req.ap_ch_report[current_channel]) + ",";
```
I think it's better to be consistent.

Reply:
The mechanism here is a bit different:
I am building the string to be placed _after_ `ap_ch_report=` and then I apply this value to `cmd` together with the `ap_ch_report=` prefix.
The structure is: `cmd += "ap_ch_report="` + `<built list>`
So the only question left is if to build the list using `stringstream operator<<` or `string operator +`. I find the former easier.